cyhy/core/geoloc.py

```diff
@@ -9,7 +9,37 @@
 GEODB_FILES = ["GeoIP2-City.mmdb", "GeoLite2-City.mmdb"]
 GEODB_CITY_PATHS = ["/usr/share/GeoIP/", "/usr/local/share/GeoIP/"]
 GEODB_FILE_PATHS = []
-RESTRICTED_COUNTRIES = ["China", "Iran", "North Korea", "Russia"]
+
+# The MaxMind GeoLite2 and GeoIP2 databases that we support use 
+# https://www.geonames.org/countries/ as the data source for country names.
+# For more info, see: 
+# https://support.maxmind.com/hc/en-us/articles/4414877149467-IP-Geolocation-Data#h_01FRRNFD5Z5EWNCAXM6SZZ5H2C
+RESTRICTED_COUNTRIES = [
+    "Afghanistan",
+    "Belarus",
+    "Cambodia",
+    "Central African Republic",
+    "China",
+    "Cuba",
+    "Cyprus",
+    "DR Congo",
+    "Eritrea",
+    "Ethiopia",
+    "Haiti",
+    "Iran",
+    "Iraq",
+    "Lebanon",
+    "Libya",
+    "Myanmar",
+    "North Korea",
+    "Russia",
+    "Somalia",
+    "South Sudan",
+    "Sudan",
+    "Syria",
+    "Venezuela",
+    "Zimbabwe"
```

Comment:
Preferred practice on Python lists.

```suggestion
    "Zimbabwe",
```